### ISSUE TITLE
Default Content

### ISSUE THREAD

**Issue by Syncaidius** [Syncaidius/MoltenEngine#68]
Implement the following on ContentManager:
- [ ] 	T asset = ContentManager.Default<T>() for retrieving the default asset of a specific type. E.g. Font or texture
- [ ] ContentManager.Default<T>(T value) for setting default asset of a specific type

Implement the following on IContentProccessor:
- [ ] IContentProccessor.LoadDefault()  - Virtual which can optionally provide a default asset, which should only be loaded on the first request of default or if a Load() request fails to find the requested file

**Comment by IceReaper** [comment]
Would a default content also allow the engine to do some lazy loading? Like requesting the asset, and using a dummy asset while is not loaded yet? While there are required assets to be able to display a scene and play, like the map geometry, some games actually allow you to play before the whole game is downloaded yet. In that case placeholder assets could be used till the real asset has been made available and processed.

**Comment by Syncaidius** [comment]
Yes, that is the intention behind new content handle system. You receive a handle to the content asset you requested and can track its status. A placeholder will be stored in it while the asset loads.

This will be next on the list once I'm done refining the font/text rendering 🙂
